src/e2e/pages/dashBoardPage.ts

```diff
@@ -77,9 +77,7 @@ export class DashboardPage {
       name: "View all sites where your info is exposed",
     });
 
-    this.exposuresCardHeading = page.getByRole("heading", {
-      name: "Let’s keep protecting your data",
-    });
+    this.exposuresCardHeading = page.getByRole("link", { name: "Home" });
```

Comment:
How is this the exposure card heading?

Reply:
its not, this is to confirm a successful redirect/login by verifying certain element in the dashboard

Reply:
So is this still a `*CardHeading` or is a `*NavButton` or `*Link` now?

Reply:
good catch, updated the name to be more appropriate here